source/treeInterceptorHandler.py

```diff
@@ -126,6 +126,15 @@ def _set_passThrough(self, state):
 					review.setCurrentMode('document',True)
 			braille.handler.handleGainFocus(self)
 
+	def _get_isSelectionAnchoredAtStart(self):

```

Comment:
1. This should be in DocumentTreeInterceptor, since text is not valid for non-document TreeInterceptors. Only DocumentTreeInterceptors have makeTextInfo.
2. Same suggested changes as for this property in NVDAObject.